OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -176,10 +176,30 @@ void UpdateNetWorth(string oldResourceType, int oldDensity, string newResourceTy
 		{
 			// Density + 1 as workaround for fixing ResourceLayer.Harvest as it would be very disruptive to balancing
```

Comment:
As you can see there's a comment that will need updating. I traced it to #13318. It seems like a bigger problem